feat(tribe-tooltips): Add Tooltip object and views. (#978)
moderntribe/tribe-common: readme.txt

== Changelog ==

= [4.9.6] TBD =

> * Tweak - Add new tribe-tooltip object for universal utility [120856]

Reviewer: ```suggestion
* Tweak - Add new `tooltip.view` PHP class to render new tool tips that utilize the `tribe-tooltip` CSS class for universal utility [120856]
```

== Changelog ==

> = [4.9.4.1] TBD =

Reviewer: ```suggestion
= [4.9.6] TBD =

```

Author: Yeah, that version number was from when I thought this was just going to be a style tweak.

* Tweak - Add new tribe-tooltip object for universal utility [120856]

> 

Reviewer: ```suggestion

= [4.9.5] TBD =

```
